plugins/plugin-template/webpack.config.js

```diff
@@ -11,7 +11,11 @@ module.exports = {
       filename: "[name].js.map",
     }),
     new CopyWebpackPlugin({
-      patterns: [{ from: "src/static", to: "static" }],
+      patterns: [{ from: "src/static", to: "static" }, {
+        from: "src/*/**.template.ts", to({ context, absoluteFilename, }) {
+          return path.join(__dirname, 'dist', absoluteFilename.replace(/^.*[\\\/]/, ''));
```

Comment:
Let's keep in mind that dropping the folder structure, when more then 1 template file is copied over the `dist/` folder, only one version for that filename will be kept. 
Not sure how often this will happen but we should at least make it more resilient to names